src/App.tsx

```diff
@@ -2,35 +2,46 @@ import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { SearchField } from './components/SearchField';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  changeQuery = (newQuery: string) => {
+    this.setState({ query: newQuery });
+  };
+
+  filteredMovies = () => {
+    const { query } = this.state;
+
+    return (
+      moviesFromServer.filter(movie => (
+        `${movie.title} ${movie.description}`
+          .toLowerCase()
+          .includes(query.toLowerCase())
```

Comment:
query.toLowerCase() better set to a separate variable not to calculate it on every iteration